### PR TITLE
Fix build: update msbuild path

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

File: azure-pipelines.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ variables:
   - group: sonarsource-build-variables
   # To make sure we don`t use MsBuild 17.
   - name: MsBuildPath
-    value: 'C:\Program Files\Microsoft Visual Studio\2022\Community\Msbuild\Current\Bin\msbuild.exe'
+    value: 'C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin\MSBuild.exe'
```
Comment (martin-strecker-sonarsource) [inline azure-pipelines.yml:20]: Please leave a comment to revert the change. The best thing to do here is IMO what is described here: https://github.com/microsoft/vswhere/wiki/Find-MSBuild

**Review by martin-strecker-sonarsource — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply azure-pipelines.yml:20]: We will not revert this change. Please add an issue if you want to refactor the scripts and use Find-MSBuild.
